python/cuml/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -260,8 +260,13 @@ class RandomForestRegressor(BaseRandomForestModel,
 
     _cpu_estimator_import_path = 'sklearn.ensemble.RandomForestRegressor'
 
+    _default_split_criterion = "mse"
+
     _hyperparam_interop_translator = {
-        "criterion": "NotImplemented",
+        "criterion": {
+            "friedman_mse": "NotImplemented",
+            "absolute_error": "NotImplemented",
```

Comment:
```suggestion
            "absolute_error": "NotImplemented",
            "squared_error": "mse",
```
So that `RandomForestRegressor()` and `RandomForestRegressor(criterion="squared_error")` both work. Explicitly passing the constructor arg's default value might happen during grid searching

Reply:
Check out the latest implementation. I think this may have been based on https://github.com/rapidsai/cuml/pull/6363/commits/4c8a80b36c3707a1ad127a58b4acff7da448c137, but the original intent was that this translation should _not_ be necessary. I believe that is true in the current implementation.